client/blocks/product-selector/index.jsx

```diff
@@ -156,7 +156,7 @@ export class ProductSelector extends Component {
 	}
 
 	getSubtitleByProduct( product ) {
-		const { currentPlanSlug, selectedSiteSlug, translate } = this.props;
+		const { currentPlanSlug, selectedSiteSlug, translate, moment } = this.props;
```

Comment:
Uber nit here: these props were ordered alphabetically before. Would be nice to maintain that.

Reply:
Fixed!